mypy/main.py

```diff
@@ -205,14 +205,15 @@ def add_invertible_flag(flag: str,
             strict_flag_assignments.append((dest, not default))
 
     def disallow_any_argument_type(raw_options: str) -> List[str]:
-        current_options = raw_options.split(',')
-        for option in current_options:
-            if option not in valid_disallow_any_options:
-                formatted_opts = ', '.join(map("'{}'".format, valid_disallow_any_options))
-                message = "Unrecognized option '{}' (valid options are: {}).".format(
-                    option, formatted_opts)
+        options = raw_options.split(',')
+        for option in options:
+            if option not in disallow_any_options:
+                formatted_valid_options = ', '.join(
+                    "'{}'".format(option) for option in disallow_any_options)
```

Comment:
`option` here shadows `option` defined on line 209 -- you really don't want to do that.  I think a one-letter `o` would be fine as a name here (because the context you need to see what that means is all right on the same line).